### PR TITLE
Remove access token usage

### LINKED ISSUE #182: Replace usage of `ACCESS_TOKEN` with `GITHUB_TOKEN`
## Problem

Currently we use a secret `ACCESS_TOKEN` containing @dhruvkb's PAT to run many actions. This can be replaced with the [`GITHUB_TOKEN` secret](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#about-the-github_token-secret) or [`github.token` context](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context) provided within GitHub Actions themselves.

## Description

Replace the custom secret `ACCESS_TOKEN` with the built-in `GITHUB_TOKEN`.

## Alternatives

`ACCESS_TOKEN` has the advantage that I get a lot of credit for making PRs, applying labels and moving cards in our Project  boards. I also got a Hacktoberfest T-shirt with little effort.

Just kidding, `GITHUB_TOKEN` is always better as that's one less secret to take care of (and reduces my presence from the bus factor).

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by sarayourfriend**
## Fixes

Fixes #182 by @dhruvkb 

## Description

Just a find and replace job of `secrets.ACCESS_TOKEN` to `secrets.GITHUB_TOKEN`.

## Testing Instructions

Not sure how to test.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [N/A] I added or updated tests for the changes I made (if applicable).
- [N/A] I added or updated documentation (if applicable).
- [N/A] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by sarayourfriend** [comment]
@WordPress/openverse-developers  Not sure what to do about the fact that the GITHUB_TOKEN doesn't have any write permissions on PRs from forks (at least that's how I'm understanding the docs). I believe this would break the PR auto-labeling.

**Comment by zackkrida** [comment]
Could go the route of a bot account, I created this but didn't finish setting it up: https://github.com/openverse-bot
It was during one of the outages last week.

**Comment by dhruvkb** [comment]
If I understand the docs correctly, the PR autolabelling action can use `GITHUB_TOKEN` for read/write since the workflow is defined to run on the [`pull_request_target` event](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target).

image

**Review by zackkrida — APPROVED** [review]
Based on @dhruvkb's assessment of permissions; let's see how it goes

File: .github/workflows/new_prs.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ jobs:
         with:
           project: Openverse PRs
           column: In progress
-          repo-token: ${{ secrets.ACCESS_TOKEN }}
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (dhruvkb) [inline .github/workflows/new_prs.yml:22]: `GITHUB_TOKEN` [might not have access](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token) to projects outside the repo.

File: .github/workflows/new_issues.yml [hunk 2]
```diff
@@ -16,4 +16,4 @@ jobs:
         with:
           project: Openverse
           column: Backlog
-          repo-token: ${{ secrets.ACCESS_TOKEN }}
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (dhruvkb) [inline .github/workflows/new_issues.yml:19]: `GITHUB_TOKEN` [might not have access](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token) to projects outside the repo.

File: .github/workflows/new_prs.yml [hunk 3]
```diff
@@ -31,4 +31,4 @@ jobs:
         with:
           project: Openverse PRs
           column: Needs review
-          repo-token: ${{ secrets.ACCESS_TOKEN }}
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (dhruvkb) [inline .github/workflows/new_prs.yml:34]: `GITHUB_TOKEN` [might not have access](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token) to projects outside the repo.

**Review by dhruvkb — COMMENTED** [review]
There are three cases where I think `GITHUB_TOKEN` might not have the right permissions. Rest looks good.

**Comment by zackkrida** [comment]
@dhruvkb can you switch the three project related comments to continue to use `ACCESS_TOKEN`, then? Perhaps with a todo comment? I think you can commit directly to the PR in light of @sarayourfriend's afk.

**Comment by dhruvkb** [comment]
I'll be monitoring some actions to see that nothing breaks.

**Review by dhruvkb — APPROVED** [review]

**Comment by AetherUnbound** [comment]
```
          # TODO: Switch to GITHUB_TOKEN if the project is moved to a repo
```

I'm confused by this comment, how would the project be moved to a repo?

**Comment by dhruvkb** [comment]
I meant the <a href="https://github.com/orgs/WordPress/projects/3">GitHub _**Project**_ board</a>.

**Comment by AetherUnbound** [comment]
Oh! That makes perfect sense, thank you!

**Comment by obulat** [comment]
> I meant the [GitHub _**Project**_ board](https://github.com/orgs/WordPress/projects/3), which is currently at the WordPress org level and not at a repo level. We could move it within WordPress/openverse, if and when we converge to a monorepo.

It seems that all projects are on the org level. Even if you click on "New project" from the Openverse repository, you get redirected to https://github.com/orgs/WordPress/projects/xx . @dhruvkb, do you think it makes sense to remove the TODO comments about switching to GITHUB_TOKEN?

**Pull request merged**